templates/default/css/template.css

```diff
@@ -654,15 +655,24 @@ div.yes-no .btn-group label {
   .table-bordered{
     border:none;
   }
-
-  .ls-answers tr:last-child{
+  
+  .ls-answers tbody:last-child tr:last-child{
```

Comment:
Yes: repeat header. But all need margin-bottom:0; no ? (and the last one can be great with  border-radius ;) )